libcloud/common/runabove.py

```diff
@@ -55,12 +55,19 @@ class RunAboveResponse(JsonResponse):
     def parse_error(self):
         response = super(RunAboveResponse, self).parse_body()
 
+        status_code = int(self.status)
+        if status_code >= 300 and status_code < 600:
+            raise ProviderError('Unknown error. Status code: %d' % self.status,
+                                http_code=status_code)
         if response.get('errorCode', None) == 'INVALID_SIGNATURE':
```

Comment:
Which status code do they return on invalid_signature?

Just want to make sure it's `< 300` so we don't need to move this if before the one you just added.